apps/events/tests/view_tests.py

```diff
@@ -246,48 +248,54 @@ def test_attend_get(self):
         self.assertInMessages("Vennligst fyll ut skjemaet.", response)
 
     def test_attend_missing_note(self):
-        form_params = {"g-recaptcha-response": "PASSED"}
         url = reverse("attend_event", args=(self.event.id,))
 
-        response = self.client.post(url, form_params, follow=True)
+        response = self.client.post(url, self.dummy_form, follow=True)
 
         self.assertRedirects(response, self.event.get_absolute_url())
         self.assertInMessages("Du må fylle inn et notat!", response)
 
     def test_attend_not_accepted_rules(self):
-        form_params = {"g-recaptcha-response": "PASSED"}
         url = reverse("attend_event", args=(self.event.id,))
         G(
             Membership,
             username=self.user.ntnu_username,
             expiration_date=timezone.now() + timedelta(days=1),
         )
 
-        response = self.client.post(url, form_params, follow=True)
+        response = self.client.post(url, self.dummy_form, follow=True)
 
         self.assertRedirects(response, self.event.get_absolute_url())
         self.assertInMessages("Du må godta prikkereglene!", response)
 
-    @patch("captcha.fields.client.submit")
-    def test_attend_invalid_captcha(self, mocked_submit):
-        mocked_submit.return_value = RecaptchaResponse(is_valid=False)
-        url = reverse("attend_event", args=(self.event.id,))
-        form_params = {"g-recaptcha-response": "WRONG"}
+    def test_attend_invalid_captcha(self):
+        # validation THROWS, it does not return a boolean
+        # therefoer we dont even overwrite it
+
+        event = G(Event)
+        G(
+            AttendanceEvent,
+            event=event,
+            registration_start=timezone.now() - timedelta(days=1),
+            registration_end=timezone.now() + timedelta(days=1),
+        )
+        url = reverse("attend_event", args=(event.id,))
+
         G(
             Membership,
             username=self.user.ntnu_username,
             expiration_date=timezone.now() + timedelta(days=1),
         )
         MarkRuleSet.accept_mark_rules(self.user)
 
-        response = self.client.post(url, form_params, follow=True)
+        response = self.client.post(url, self.dummy_form, follow=True)
 
-        self.assertRedirects(response, self.event.get_absolute_url())
-        self.assertInMessages("Du klarte ikke captchaen! Er du en bot?", response)
+        self.assertRedirects(response, event.get_absolute_url())
+        self.assertInMessages("Vennligst vis at du er human.", response)
```

Comment:
Dette er jo to forskjellige feil, "Du klarte ikke" vs "Du glemte" (nå "vennligst vis"), aka forskjellen mellom at captcha-responsen er med eller om innholdet er ugyldig, slik jeg forstår skal denne testen være for at den er ugyldig, men du har endret den til å teste at mangler?

Reply:
Det er den tidligere som viste feil, required og invalid var satt til samme streng

<img width="642" alt="image" src="https://github.com/dotkom/onlineweb4/assets/24441708/7a8efaf4-bd5f-4964-b68a-60838b527ceb">

Reply:
fiksa også den her, står ikke human lenger men heller den nye fiksa